DEPENDENCY-LICENSES

```diff
@@ -455,33 +464,33 @@ List of third-party dependencies grouped by their license type.
 
     Apache License, Version 2.0, Eclipse Public License - Version 1.0
 
-        * Apache :: JSTL module (org.eclipse.jetty:apache-jstl:9.4.45.v20220203 - http://tomcat.apache.org/taglibs/standard/)
-        * Jetty :: Apache JSP Implementation (org.eclipse.jetty:apache-jsp:9.4.45.v20220203 - https://eclipse.org/jetty/apache-jsp)
-        * Jetty :: Asynchronous HTTP Client (org.eclipse.jetty:jetty-client:9.4.45.v20220203 - https://eclipse.org/jetty/jetty-client)
-        * Jetty :: Continuation (org.eclipse.jetty:jetty-continuation:9.4.45.v20220203 - https://eclipse.org/jetty/jetty-continuation)
-        * Jetty :: Http Utility (org.eclipse.jetty:jetty-http:9.4.45.v20220203 - https://eclipse.org/jetty/jetty-http)
-        * Jetty :: IO Utility (org.eclipse.jetty:jetty-io:9.4.45.v20220203 - https://eclipse.org/jetty/jetty-io)
-        * Jetty :: JAAS (org.eclipse.jetty:jetty-jaas:9.4.45.v20220203 - https://eclipse.org/jetty/jetty-jaas)
-        * Jetty :: JNDI Naming (org.eclipse.jetty:jetty-jndi:9.4.45.v20220203 - https://eclipse.org/jetty/jetty-jndi)
-        * Jetty :: Plus (org.eclipse.jetty:jetty-plus:9.4.45.v20220203 - https://eclipse.org/jetty/jetty-plus)
-        * Jetty :: Rewrite Handler (org.eclipse.jetty:jetty-rewrite:9.4.45.v20220203 - https://eclipse.org/jetty/jetty-rewrite)
+        * Apache :: JSTL module (org.eclipse.jetty:apache-jstl:9.4.53.v20231009 - http://tomcat.apache.org/taglibs/standard/)
+        * Jetty :: Apache JSP Implementation (org.eclipse.jetty:apache-jsp:9.4.53.v20231009 - https://eclipse.org/jetty/apache-jsp)
+        * Jetty :: Asynchronous HTTP Client (org.eclipse.jetty:jetty-client:9.4.53.v20231009 - https://eclipse.org/jetty/jetty-client)
+        * Jetty :: Continuation (org.eclipse.jetty:jetty-continuation:9.4.53.v20231009 - https://eclipse.org/jetty/jetty-continuation)
+        * Jetty :: Http Utility (org.eclipse.jetty:jetty-http:9.4.53.v20231009 - https://eclipse.org/jetty/jetty-http)
+        * Jetty :: IO Utility (org.eclipse.jetty:jetty-io:9.4.53.v20231009 - https://eclipse.org/jetty/jetty-io)
+        * Jetty :: JAAS (org.eclipse.jetty:jetty-jaas:9.4.53.v20231009 - https://eclipse.org/jetty/jetty-jaas)
+        * Jetty :: JNDI Naming (org.eclipse.jetty:jetty-jndi:9.4.53.v20231009 - https://eclipse.org/jetty/jetty-jndi)
+        * Jetty :: Plus (org.eclipse.jetty:jetty-plus:9.4.53.v20231009 - https://eclipse.org/jetty/jetty-plus)
+        * Jetty :: Rewrite Handler (org.eclipse.jetty:jetty-rewrite:9.4.53.v20231009 - https://eclipse.org/jetty/jetty-rewrite)
```

Comment:
I guess it's a safe choice for releasing 2.6.0 soon - jumping to jetty 12.x will probably require more time (likely migration to Jakarta EE, etc)

Reply:
We have a lot of "old" dependencies, which will likely **not** support jakarta.* out of the box. So I guess, that it will take some more days until we are fully jakarta.* ready on the storm side of things. Guess the webapp is the simpliest thing to migrate.